examples/hacker_news_pwa/firebase.json

```diff
@@ -15,8 +15,14 @@
     "headers": [
       {
         "source": "/",
-        "headers": [{"key": "Link", "value": "</main.dart.js>;rel=preload;as=script"}]
+        "headers": [{
+          "key": "Link",
+          "value": "</main.dart.js>;rel=preload;as=script"
+        }, {
+          "key" : "Cache-Control",
+          "value" : "max-age=86400"
+        }]
       }
-    ]
+    ],
```

Comment:
Will trailing common break JSON parsers? Tis not Dart code 😄

Reply:
Good catch, done.